lib/elements/element.js

```diff
@@ -52,7 +52,21 @@ class Element {
       returnByValue: true,
     });
     if (result.value === undefined) {
-      throw new Error(`${this.description} does not have disable attribute`);
+      return false;
+    }
+    return result.value;
+  }
+
+  async isDraggable() {
```

Comment:
Can we document this API?

Reply:
It has to be exposed from element wrapper also then, will do